internal/task/task/checker/network.go

```diff
@@ -82,19 +130,31 @@ func NewCheckPortInUseTask(curveadm *cli.CurveAdm, dc *topology.DeployConfig) (*
 		dc.GetHost(), dc.GetRole(), joinPorts(dc, addresses))
 	t := task.NewTask("Check Port In Use <network>", subname, hc.GetSSHConfig())
 
-	var out string
-	var success bool
+	t.AddStep(&step.PullImage{
+		Image:       dc.GetContainerImage(),
+		ExecOptions: curveadm.ExecOptions(),
+	})
+
+	var containerId string
+	ss_template := "ss"
+	ss_template += " --listening"
+	ss_template += " --no-header"
+	ss_template += " --tcp '" + FORMAT_FILTER_SPORT + "'"
```

Comment:
```suggestion
cli := ctx.Module().Shell().SocketStatistics("")
cli.AddOption("--no-header")
cli.AddOption("--processes")
cli.AddOption("--listening")
command, err := cli.String()
if err != nil {
    return nil, err
}
```

Reply:
`CreateContainer` is wrapped by `step2InitCheckPort`, then use ctx.Module().Shell().SocketStatistics.